namespaces/_default_.yml

```diff
@@ -51,17 +51,17 @@ _default_:
       fields:
         - name: raw
           ignore_above: 256
-          type: string
+          type: keyword
       norms:
```

Comment:
I was getting errors when I tried deploying on ES5 - it complained that norms don't work for ES5, did you see that?

Reply:
I will check that ...

Reply:
I think there is no value in using norms with `ip` fields. Let's remove this from the `_default_.yml` for the `name: ipaddr4` field. I assume this is the same as using `norms: false` in this case.

Also [online guide suggests](https://www.elastic.co/guide/en/elasticsearch/guide/master/scoring-theory.html#field-norm) that for logging use case it is probably better to turn off norms.

> For use cases such as logging, norms are not useful. All you care about is whether a field contains a particular error code or a particular browser identifier. The length of the field does not affect the outcome. Disabling norms can save a significant amount of memory.